index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tic Tac Toe</title>
+    <meta charset="utf-8">
+     <link rel="stylesheet" type="text/css" href="style.css"/>
+  </head>
+
+  <body>
+    <table id = "playField" border="1">
+      <tr class = "row1">
+        <td class = "cell button"> X </td>
+        <td class = "cell button"> X </td>
+        <td class = "cell button"> X </td>
+      </tr>
+
+      <tr class = "row2">
+        <td class = "cell button" > X </td>
```

Comment:
I'd recommend not putting spaces around `=`. It can actually make things break in some browsers!